app/models/reaction.rb

```diff
@@ -79,6 +79,26 @@ def skip_notification_for?(receiver)
       (receiver.is_a?(User) && reactable.receive_notifications == false)
   end
 
+  def vomit_on_user?
+    reactable_type == "User" && category == "vomit"
+  end
+
+  def reaction_on_organization_article?
+    reactable_type == "Article" && reactable.organization.present?
+  end
+
+  def target_user
+    if reactable_type == "User"
+      reactable
+    else
+      reactable.user
+    end
+  end
+
+  def negative?
+    category == "vomit" || category == "thumbsdown"
+  end
```

Comment:
should all of these be unit tested or do you think it's superflous?

Reply:
I think these don't need additional testing, personally.